### ISSUE TITLE
[SUGGESTION] multiplatform version

### ISSUE THREAD

**Issue by forart** [fifonik/FFMetrics#93]
Hi there, this project is really cool !

Anyway we would suggest you to take in serious consideration to make it multiplatform (Linux, MacOS, etc.) compatible.

Maybe you could "get inspired" by @cdgriffith 's [FastFlix](https://github.com/cdgriffith/FastFlix), which has a similar interface (and is FFMPEG-based too):

![](https://raw.githubusercontent.com/cdgriffith/FastFlix/master/docs/gui_preview.png)

Hope that inspires !

**Comment by fifonik** [comment]
I do not have plans to do this, sorry.

**Comment by FazlyMR** [comment]
Hello. Sorry for commenting on an already closed issue but I have managed to launch FFmetrics on Bottles, an application to easily manage Windows applications on Linux and create sandboxed Wine environment, using Wine 8.0 as the runner.

I have not tested it fully but I have managed to launch the GUI.

**Comment by FazlyMR** [comment]
Update. I have managed to fully calculate PSNR, SSIM, and VMAF values of a compressed video against a losslessly-compressed reference with the above setup.

To reproduce it, install Bottles as a Flatpak (it includes a built-in Wine 8.0 build), then create a "Bottle" (sandboxed environment) and open FFmetrics. I also have not installed .NET Framework 4.8.0 on the sandboxed environment I used to run FFmetrics.

![Screenshot from 2023-03-17 18-31-06](https://user-images.githubusercontent.com/69378798/225866901-560ef75b-b6cb-4a82-b039-a3c99c8c0d77.png)
